web-animations-2/Overview.bs

```diff
@@ -2937,6 +2937,27 @@ new SequenceEffect(
 
 </div>
 
+<h4 id="keyframe-offset-type">Modification to the <code>*Keyframe</code> dictionaries</h4>
+
+The <code>double?</code> type of the <code>offset</code> key
+in <code>ComputedKeyframe</code>,
+<code>BaseComputedKeyframe</code>,
+<code>BaseKeyframe</code>,
+<code>BasePropertyIndexedKeyframe</code>,
+etc.
+is replaced by <code>(CSSNumberish? or TimelineRangeOffset or DOMString)</code>,
+where <code>double</code> continues to be interpreted as a percentage,
+a {{DOMString}} is parsed into a {{TimelineRangeOffset}}
+according to the syntax of the ''@keyframes'' prelude,
+and the resulting {{TimelineRangeOffset}} or {{CSSNumericValue}}
+is interpreted as defined for ''@keyframes''.
+
+If the {{DOMString}} fails to parse as a valid ''@keyframes'' prelude,
+then the keyframe is invalid,
+and will throw a <span class=exceptionname>TypeError</span>
+when [[web-animations-1#processing-a-keyframes-argument|processed]]
+(same as if the <code>doube</code> value was outside the [0,1] range).
```

Comment:
```suggestion
(same as if the <code>double</code> value was outside the [0,1] range).
```

Although actually I think we probably don't need this parenthetical?

Reply:
Added it to help link the two concepts and where they occur in the processing model, since we're not yet incorporating the full text of the processing rules. I agree the parenthetical shouldn't be needed after merging in WA1, though.

Lmk if I should remove it; I don't mind too much either way, but I'm somewhat inclined to keep it until then, since I'm worried someone could come up a "but when is it thrown" type question otherwise...

Reply:
Sure, let's leave it until merging with WA1 then.